imagebuildah/build.go

```diff
@@ -38,6 +38,9 @@ const (
 	Uncompressed = archive.Uncompressed
 )
 
+// ErrNoSymlink is returned when the path porvided is not a symbolic link
+var ErrNoSymlink = errors.New("path is not a symbolic link")
```

Comment:
This error value doesn't appear to be returned by anything.

Reply:
right, removed.